### PR TITLE
Update README's first look and adds some details about our guidelines

### PR DESCRIPTION
**Pull request by aldroydzito**
Hey, I'm updating the first look of README and adding some details about our guidelines, giving some introduction about the Code of Conduct and contributing guide 😄

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,40 @@
...
+
+<h3 align="center">
+Create ReactJS uncontrolled form structures with nested fields, validations and much more! 🚀
+</h3>
+
+<p align="center">
+
+<img alt="npm" src="https://img.shields.io/npm/v/@rocketseat/unform.svg">
+
+<img alt="dependencies" src="https://david-dm.org/@rocketseat/unform.svg">
```
Comment (diego3g) [inline README.md]: Should we use that?

File: README.md [hunk 2]
```diff
@@ -355,8 +384,16 @@ function App() {
 
 ## Contributing
 
-Before submitting a pull request, please read our [Contributing Guide](.github/CONTRIBUTING.md). It contains a set of instructions and guidelines that aim to make the contribution process easy and frictionless.
+Thanks for being interested on making this package better. We encourage everyone to help improving this project with some new features, bug fixes and performance issues. Please take a little bit of your time to read our guides, so this process can be faster and easier.
+
+### Contributing Guide
```
Comment (jpdemagalhaes) [inline README.md]: `Contributing Guide` -> `Contribution Guide`;

Or perhaps stick to the previous pattern `Contribution Guidelines`, or change all to `Contribution Guide`?

What do you guys think? Which one should we keep?

File: README.md [hunk 3]
```diff
@@ -48,7 +75,9 @@ yarn add @rocketseat/unform
     - [React Select](#react-select)
     - [React Datepicker](#react-datepicker)
 - [Contributing](#contributing)
-- [License](#license)
+  - [Contributing Guidelines](#contributing-guide)
```
Comment (jpdemagalhaes) [inline README.md]: `Contributing Guidelines` -> `Contribution Guidelines`;

File: README.md [hunk 4]
```diff
@@ -355,8 +384,16 @@ function App() {
 
 ## Contributing
 
-Before submitting a pull request, please read our [Contributing Guide](.github/CONTRIBUTING.md). It contains a set of instructions and guidelines that aim to make the contribution process easy and frictionless.
+Thanks for being interested on making this package better. We encourage everyone to help improving this project with some new features, bug fixes and performance issues. Please take a little bit of your time to read our guides, so this process can be faster and easier.
+
+### Contributing Guide
+
+Take a moment to read about our [contributing guidelines](/.github/CONTRIBUTING.md) so you can understand how to submit an issue, commiting and pull requests.
```
Comment (jpdemagalhaes) [inline README.md]: First one is a typo:
`contributing guidelines` -> `contribution guidelines`;

Second one is more of a grammar issue:
`...submit an issue, commiting and pull requests.` -> `...submit an issue, commit and create pull requests.`

File: README.md [hunk 5]
```diff
@@ -355,8 +384,16 @@ function App() {
...
+
+### Code of Conduct
+
+We expect you to follow or [Code of Conduct](/.github/CODE_OF_CONDUCT.md). Here you can read it and understand what kind of behaviour will not be tolerated.
 
-## License
+### License
```
Comment (pellizzetti) [inline README.md]: I really don't think that `License` should be a sub-item of `Contributing`. While it does play a role, the license is not limited to the scope of contributions.

File: README.md [hunk 6]
```diff
@@ -355,8 +384,16 @@ function App() {
...
-Before submitting a pull request, please read our [Contributing Guide](.github/CONTRIBUTING.md). It contains a set of instructions and guidelines that aim to make the contribution process easy and frictionless.
+Thanks for being interested on making this package better. We encourage everyone to help improving this project with some new features, bug fixes and performance issues. Please take a little bit of your time to read our guides, so this process can be faster and easier.
+
+### Contributing Guide
+
+Take a moment to read about our [contributing guidelines](/.github/CONTRIBUTING.md) so you can understand how to submit an issue, commiting and pull requests.
+
+### Code of Conduct
+
+We expect you to follow or [Code of Conduct](/.github/CODE_OF_CONDUCT.md). Here you can read it and understand what kind of behaviour will not be tolerated.
```
Comment (pellizzetti) [inline README.md]: Small typo: `or` -> `our`.

Small pet peeves:
- I feel like the "Here" is out of place, what is "Here"? It seems that should be link but the CoC itself is linked in the prior sentence.
- `behaviour will not be tolerated`-> `behaviour will and will not be tolerated`.

File: README.md (hunk 2 above)
Comment (pellizzetti) [reply README.md]: Make it `Contribution Guidelines` so it stays consistent with the `CONTRIBUTING.md` title, btw, I gotta change a similar typo there.

File: README.md (hunk 1 above)
Comment (pellizzetti) [reply README.md]: Yeah... It seems that this badge is rather unstable.

There is also the fact that being careful if all dependencies are up-to-date is a pain...

File: README.md [hunk 7]
```diff
@@ -386,14 +386,14 @@ function App() {
...
-### Contributing Guide
+### Contribution Guidelines
 
-Take a moment to read about our [contributing guidelines](/.github/CONTRIBUTING.md) so you can understand how to submit an issue, commiting and pull requests.
+Take a moment to read about our [Contribution Guidelines](/.github/CONTRIBUTING.md) so you can understand how to submit an issue, commit and pull requests.
 
 ### Code of Conduct
 
-We expect you to follow or [Code of Conduct](/.github/CODE_OF_CONDUCT.md). Here you can read it and understand what kind of behaviour will not be tolerated.
+We expect you to follow our [Code of Conduct](/.github/CODE_OF_CONDUCT.md). You can read it and understand what kind of behaviour will and will not be tolerated.
```
Comment (pellizzetti) [inline README.md]: `to understand` seems to be a better fit.

File: README.md (hunk 1 above)
Comment (aldroydzito) [reply README.md]: Fair enough, I'm removing this.

**Pull request merged**